### DISCUSSION TITLE
markmap zooming incorectly when using markmap@0.18

### DISCUSSION THREAD

**Discussion by loctran016** [vuepress-theme-hope/vuepress-theme-hope#4657] [Q&A] [answered]
Hmm, tbh I'm not sure is this is an bug of markmap or vuepress-theme-hope (cuz markmap have some breaking changes)

Initially, when you load the page, the markmap will display incorrectly:
![image](https://github.com/user-attachments/assets/c79dbef5-8f45-44e7-be12-da9617213d35)

Then you chang page zoom and it display correctly:
![image](https://github.com/user-attachments/assets/20eb7814-8c1c-4233-a9e4-8231fb9775fd)

Though this seem doesn't relate to viewport (zoom percentage) of the website, cuz reload the page (still zoom 90%) and it displays incorectly

![image](https://github.com/user-attachments/assets/559a2e92-1e42-409b-a826-52cd07baf5ac)

**Comment by loctran016** [comment]
And yes, revert it back to 0.17.2 does fix this

**Comment by Mister-Hope** [comment]
We do not support 0.18 yet

**Reply by loctran016** [reply]
Yes, that's why I put it here for someone who experienced the same issue :)))))))))))))))

**Comment by Mister-Hope** [comment, marked as answer]
This is now supported
